### ISSUE TITLE
SQS recreate fails due to required AWS timeout

### ISSUE THREAD

**Issue by hashibot** [terraform-providers/terraform-provider-aws#831]
_This issue was originally opened by @batkuip as hashicorp/terraform#15139. It was migrated here as part of the [provider split](https://www.hashicorp.com/blog/upcoming-provider-changes-in-terraform-0-10/). The original body of the issue is below._

<hr>

### Terraform Version
0.9.6

### Affected Resource(s)
aws_sqs_queue

### Terraform Configuration Files
```hcl
resource "aws_sqs_queue" "etl_deadletter" {
  name = "${var.prefix}-etl-deadletter"
}
```

### Debug Output
```
* aws_sqs_queue.etl_deadletter: Error creating SQS queue: AWS.SimpleQueueService.QueueDeletedRecently: You must wait 60 seconds after deleting a queue before you can create another with the same name.
```

### Expected Behavior
Recreation of the queue

### Actual Behavior
Blocked due to 60 seconds required delay on AWS side

### Steps to Reproduce
1. `terraform apply`

### Important Factoids
Tried to use 'name-prefix' as used by others but that wouldn't work for sqs queue.  That would be a nice easy fix.

**Comment by radeksimko** [comment]
Just reopened per discussion in #855

**Comment by bflad** [comment]
I'll be submitting an acceptance test and associated resource update for this very shortly. 👍

**Comment by bflad** [comment]
The fix for this has been merged into master and will release with v1.8.0 of the provider, expected tomorrow! 🎉

**Comment by bflad** [comment]
This has been released in [terraform-provider-aws version 1.8.0](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#180-january-29-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
